### PR TITLE
he monke

### PR DESCRIPTION
**Pull request by Cenrus**
## About this PR

increases monky cap

## Why it's good for the game

it is not. god help us.

## Changelog

:cl:
add: adds monkey
/:cl:

### CODE REVIEW

**Comment by enthethan** [comment]
****NO GODS ONLY MONKE****

**Comment by Oshibkka** [comment]
cenrus if this passes unlike my monkey pr im going to cry

File: config/Sage/game_options.txt [hunk 1]
```diff
@@ -564,7 +564,7 @@ RANDOMIZE_SHIFT_TIME
 #SHIFT_TIME_REALTIME
 
 ## A cap on how many monkeys may be created via monkey cubes
-MONKEYCAP 64
+MONKEYCAP 150
```
Comment (Crossedfall) [inline config/Sage/game_options.txt]:
```suggestion
MONKEYCAP 100
```

Maybe keep Sage down to 100 or so. How often does it even hit the 64 cap we currently have?

Comment (Cenrus) [reply config/Sage/game_options.txt]: I've never seen anyone meet the cap.

**Pull request merged**